internal/sbom/spdx.go

```diff
@@ -125,6 +170,151 @@ func GenerateSPDX(koVersion string, date time.Time, mod []byte, imgDigest v1.Has
 	return buf.Bytes(), nil
 }
 
+func extractDate(sii oci.SignedImageIndex) (*time.Time, error) {
```

Comment:
Instead of extracting the date from the index we just built, can we pass the created time from gobuild.go into where we generate the SBOMs?

Or is this just to support `ko deps`?

Reply:
That's what the image SBOM logic was doing, but extracting it from the image's config, which is frankly more accessible.  I'll take a quick look at how hard it would be to pass it in without diverging the source of truth logic for setting these timestamps.

Reply:
Looks doable, lemme plumb that through instead. 👍

Reply:
Done!

Reply:
Ugh, so it looks like we only use `g.creationTime` under certain circumstances, and we let the base image's creation time through otherwise, so for the index SBOM I think we'll end up needing some sort of extraction method to make that really work.

Reply:
e.g. 
```
	empty := v1.Time{}
	if g.creationTime != empty {
		cfg.Created = g.creationTime
	}
```

Reply:
I'm reverting to what I had, as I think it's cleanest modulo a bigger refactor of this code.